include/options.hxx

```diff
@@ -545,9 +545,7 @@ public:
   bool isValue() const {
     return is_value;
   }
-  bool isSection() const {
-    return is_section;
-  }
+  bool isSection(const std::string name = "") const;
```

Comment:
You probably want to take `std::string` by `const&`, unless you're going to take a copy in the function anyway